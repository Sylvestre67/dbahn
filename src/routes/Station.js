import React from 'react';
// import PropTypes from 'prop-types';

import StationHeader from '../components/StationHeader';
import StationTimetable from '../components/StationTimetable';

class Station extends React.Component {
	render() {
		return (
			<React.Fragment>
				<StationHeader />
				<StationTimetable />
			</React.Fragment>
		);
	}
}

export default Station;
