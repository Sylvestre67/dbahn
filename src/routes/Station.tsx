import React from 'react';

import StationHeader from '../components/StationHeader';
import StationTimetable from '../components/StationTimetable';

import apolloClient from '../services/apollo';

export const Station: React.SFC<{}> = () => {
	return (
		<React.Fragment>
			<StationHeader client={apolloClient} />
			<StationTimetable client={apolloClient} />
		</React.Fragment>
	);
};

export default Station;
