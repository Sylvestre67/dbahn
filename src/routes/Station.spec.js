import React from 'react';
import { shallow } from 'enzyme';

import Station from './Station';
let component;

const props = {};

describe('Station', () => {
	beforeAll(() => {
		component = shallow(<Station {...props} />);
	});

	it('should render correctly', () => {
		expect(component).toMatchSnapshot();
	});
});
