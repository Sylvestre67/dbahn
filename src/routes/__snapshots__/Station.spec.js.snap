// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Station should render correctly 1`] = `
<Fragment>
  <WithStyles(withApollo(withRouter(StationHeader))) />
  <WithStyles(withRouter(withApollo(StationTimetable))) />
</Fragment>
`;
